### DISCUSSION TITLE
Revisions for Quizzes and Homeworks

### DISCUSSION THREAD

**Discussion by AjeethUConn** [cooperrc/advanced-dynamics#27] [Logistics]
I thought that I read on the syllabus before the course started that there were revisions for quizzes and homework. Has that policy been changed, or am I looking in the wrong place?

**Comment by SentientBiped** [comment]
I believe he said in the [intro video](https://www.youtube.com/watch?v=cIsFmVVtmYE) that homework can be revised at any time but that quizzes are one and done. You may or may not need to let him or the TA know of your resubmission, too. That I don't know.
